CHANGES.rst

```diff
@@ -1,6 +1,21 @@
 Changes
 =======
 
+Version master (unreleased)
```

Comment:
We should decide on a common header. What about `Unreleased`? Capital U.

Reply:
In accordance with `FIXME` and `CHANGEME`, I'd go with `UNRELEASED` all caps.